datalad/support/annexrepo.py

```diff
@@ -852,6 +852,41 @@ def get_remotes(self, with_refs_only=False, with_urls_only=False,
         else:
             return remotes
 
+    def get_special_remotes(self):
```

Comment:
note that also now we have `is_special_annex_remote` so may be should be refactored to use this `get_special_remotes`

Reply:
Not sure about this one @yarikoptic The present `is_special_annex_remote()` is a fast implementation that requires no calls to annex. RF to use `get_special_remote()` would be more uniform, but also a lot slower. Although this function isn't used in many places, I'd say we leave it as is.